src/assets/scripts/client/info/simAirportInfoTemplate.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * HTML template for the airport info view and clock.
+ *
+ * @property SIM_AIRPORT_INFO_TEMPLATE
+ * @type {String|HTML Element}
+ * @final
+ */
+export const SIM_AIRPORT_INFO_TEMPLATE = '' +
+'<div id="airport-info">' +
+'    <div id="definitions" class="sidebyside">' +
+'        <div id="clock-def" class="vertical">TIME</div>' +
+'        <div id="wind-def" class="vertical">WIND</div>' +
+'        <div id="altimeter-def" class="vertical">ALTIM</div>' +
```

Comment:
The whole structure of these divs seems backwards to me. I would have thought it would be better to have four divs, each with a label (left side) and a value (right side). That would also help with the so-so naming situation with the use of "definitions" here. We could do
```
<div id="clock" class="airport-info-line">
    <div id="clock-label" class="airport-info-line-label">TIME |</div>
    <div id="clock-value" class="airport-info-line-value">XXXX/XX</div>
</div>
...
```

But we'll see what mr. @n8rzz says, this is definitely not my area.

Reply:
Completely agree.  And, in fact, I'd take it one step further and remove the inline pipe.  so you'd end up with something like (note new naming):

```html
<div class="airportInfo-item js-airportInfo-item-clockView">
    <div class="airportInfo-item-label js-airportInfo-item-clockView-label">TIME</div>
    <div class="airportInfo-item-value js-airportInfo-item-clockView-value">XXXX/XX</div>
</div>
```

```css
.airportInfo-item-value {
    // ...
    border-left: 1px solid;
    padding-left: 2px;  // or whatever is appropriate
    margin-left: 2px;  // or whatever is appropriate
}
```

--

Couple callouts here:
- avoid the use of css `ids`, instead use a dedicated `js-` prefixed _class name_ that should be used in the view class. (again, see `StripViewModel` for examples)
- the html and css shouldn't need to care what is being displayed, each label/value div should be data agnostic
- using the `js-` prefixes alleviates the need for those ids

Reply:
done